pb/summary/summary.proto

```diff
@@ -45,6 +47,74 @@ message FailingTestSummary {
   string fail_test_link = 13;
 }
 
+// Represents the number of tests that have a flakiness at or above the threshold.
+// If a test is counted in this bucket for this threshold, it will not be counted
+// for buckets of a lower threshold.
+//
+// E.g. if a test has a flakiness of 50.0, and there are two buckets: threshold 20.0
+// and threshold 3.0, the test will be counted for the 20.0 bucket and not the 3.0
+// one, even though the flakiness is above both thresholds.
+message FlakyBucket {
+  // Minimum flakiness for the bucket
+  float threshold = 1;
+  // number of tests in the bucket
+  int32 tests = 2;
+}
+
+// Metrics about a specific test, i.e. passes, fails, total runs, etc.
+message TestInfo {
+  // Test name.
+  string name = 1;
+  // Test environment, i.e. staging-1.13
+  string env = 2;
+
+  int32 total_runs = 3;
+  int32 passed_runs = 4;
+  int32 failed_runs = 5;
+  int32 failed_infra_runs = 6;
```

Comment:
are passed_infra_runs not needed? And I am not sure what the standard is on organizing fields but I think it would be nice to make `total_runs_with_infra = 7` just so like things are close

Reply:
I think here `total_runs = passed_runs + failed_runs + failed_infra_runs + other_runs`, we can probably rename `failed_runs` to `failed_non_infra_runs`?

Reply:
I moved up total_runs_with_infra. Regarding passed_infra_runs: I don't think that's actually a thing. failed_infra_runs refers to tests that have failed due to infrastructure failures I believe. Specifically, it's categorized by a failed test run with an accompanying message, something like "SOME_INFRA_FAILURE". I guess the naming is a bit confusing but I've updated the comments for all the fields so hopefully it becomes a bit more clear with that.